gallery/others/plot_optical_flow.py

```diff
@@ -3,6 +3,10 @@
 Optical Flow: Predicting movement with the RAFT model
 =====================================================
 
+.. note::
+    Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_optical_flow.ipynb>`_
```

Comment:
Note that all links point to `main`. I don't know how to point to the current version, not sure it's worth it.

I think that in the tutorials repo they literally have a separate `ipynb` file which always get outdated... That sounds a lot worse.